osu.Desktop/Program.cs

```diff
@@ -48,16 +57,10 @@ public static int Main(string[] args)
                         return 0;
                 }
 
-                switch (args.FirstOrDefault() ?? string.Empty)
```

Comment:
Why is this changed from a `switch`?

Reply:
Because there can now be more than one argument, I've changed this to use `args.Contains("--tournament")` rather than assuming the first argument will always have that.

Reply:
That's not how it should work. At *very* least a naive approach would split and process the `args`, handling via a `switch` once at a time.

Reply:
I don't quite understand what you mean?

Reply:
```csharp
foreach (string arg in args)
{
    switch(arg)
    {
        // handle each arg.
    }
}
```